src/jdk.jlink/linux/classes/jdk/tools/jlink/internal/plugins/StripNativeDebugSymbolsPlugin.java

```diff
@@ -131,6 +136,33 @@ public ResourcePool transform(ResourcePool in, ResourcePoolBuilder out) {
         return out.build();
     }
 
+    /**
+     * Method to determine if a particular resource should be stripped.
```

Comment:
This javadoc suggests what I think is a better name for the method: `shouldStrip()`.

Reply:
Renamed the method to `shouldStrip()`